### PR TITLE
Style export box (Meta #116)

### PR DESCRIPTION
**Pull request by marcwieland95**

### CODE REVIEW

File: src/Resources/views/tailwind_2/_header.html.twig [hunk 1]
```diff
@@ -28,27 +28,26 @@
                         data-transition-leave-to="opacity-0 scale-95"
                         tabindex="-1"
                     >
-                        <div class="space-y-2" {% if table.exporters|length > 1 %} {{ stimulus_controller('araise/table-bundle/exporter') }} {% endif %}>
+                        <div {% if table.exporters|length > 1 %} {{ stimulus_controller('araise/table-bundle/exporter') }} {% endif %}>
```
Comment (renestalder) [inline src/Resources/views/tailwind_2/_header.html.twig:31]: Putting If-Elses inline into a HTML Tag is still a bad idea for any sorts of formatting. We need to use other approaches. This is not maintainable and sustainable.

Instead, we could but the stimulus controller on it anyway, and disable it with a stimulus controller value attribute. Much cleaner to read, with no drawbacks.

File: src/Resources/views/tailwind_2/_header.html.twig [hunk 2]
```diff
@@ -28,27 +28,26 @@
...
+                                <ul class="mb-4" role="list">
+                                    {% for acronym,exporter in table.exporters %}
+                                        <li class="flex items-center mb-2 gap-x-2">
+                                            <input
+                                                class="whatwedo_core-checkbox"
+                                                type="checkbox"
+                                                id="{{ acronym }}"
+                                                name="exporter"
+                                                value="{{ acronym }}"
+                                                {% if table.exporters|length > 1 %} {{ stimulus_action('araise/table-bundle/exporter', 'select')}} {% endif %}
```
Comment (renestalder) [inline src/Resources/views/tailwind_2/_header.html.twig:43]: Same with the inline if-else. We need to find another way.

**Review by renestalder — CHANGES_REQUESTED** [review]

File: src/Resources/views/tailwind_2/_header.html.twig (hunk 1 above)
Comment (marcwieland95) [reply src/Resources/views/tailwind_2/_header.html.twig:31]: @renestalder I'm against loading Stimulus Controller which are not needed. Those should not be initiated at all if unused.

The same as we should use `html_classes`, we should also do something similar with attributes and following Stimulus Controllers. Although I'm not sure if something like that exists.

File: src/Resources/views/tailwind_2/_header.html.twig (hunk 2 above)
Comment (marcwieland95) [reply src/Resources/views/tailwind_2/_header.html.twig:43]: I just did a little improvement by moving the if/else statement out of the html element.

In the future a helper could be created like this:

```
{% set attributes = {
    'id': acronym,
    'name': 'exporter',
    'value': acronym,
} %}
<input{{ html_attributes(attributes) }}>
```

Out of scope now. But noted

File: src/Resources/views/tailwind_2/_header.html.twig (hunk 1 above)
Comment (renestalder) [reply src/Resources/views/tailwind_2/_header.html.twig]: @marcwieland95 Well, me too, but I guess the current setup is anyway creating one big chunk which is already loaded by the browser, we would have saved nothing from the conditional applying of the controller.

However, with the refactoring now, the feedback is obsolete.

**Review by renestalder — APPROVED** [review]
@marcwieland95 lgtm

**Pull request merged**